src/web/lib/components/ExportThemeDialog/index.js

```diff
@@ -1,27 +1,113 @@
 import React from "react";
-import "./index.scss";
+import Modal from "../Modal";
 
-export const ExportThemeDialog = ({
-  isThemeExportInProgress,
-  shouldOfferExportedTheme,
-  exportedTheme,
-  clearExportedTheme
-}) => (
-  <div>
-    {isThemeExportInProgress && <div>THEME EXPORT IN PROGRESS</div>}
-    {shouldOfferExportedTheme && (
-      <div>
-        Exported theme:{" "}
-        <a download="theme.zip" href={exportedTheme}>
-          theme.zip
-        </a>{" "}
-        <a download="theme.xpi" href={exportedTheme}>
-          theme.xpi
-        </a>{" "}
-        <button onClick={clearExportedTheme}>Clear</button>
-      </div>
-    )}
-  </div>
-);
+class ExportThemeDialog extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      panelIndex: 0
+    };
+  }
+
+  handleNext() {
+    let panel = this.state.panelIndex;
+    panel++;
+    this.setState({ panelIndex: panel });
+  }
+
+  handleClose() {
+    this.setState({panelIndex: 0});
+    this.props.clearExportedTheme();
+  }
+
+  renderContent() {
+    const { panelIndex } = this.state;
+    const { exportedTheme } = this.props;
+
+    switch (panelIndex) {
+      case 0:
+        return (
+          <>
+            <h2>Export your theme</h2>
+            <p>
+              Create a compressed version of your theme that you can submit to
+              the{" "}
+              <a
+                href="https://addons.mozilla.org/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Firefox Add-ons Marketplace
+              </a>
+              .
+            </p>
+            <button className="modal__button" onClick={() => this.handleNext()}>
+              Next
+            </button>
+          </>
+        );
+      case 1:
+        return (
+          <>
+            <h2>Name your theme</h2>
+            <form className="modal__form">
+              <label>Give your theme a descriptive name.</label>
+              <input type="text" maxLength="32" placeholder="Theme name" />
+            </form>
+            <button className="modal__button" onClick={() => this.handleNext()}>
```

Comment:
Having the button inside the form with `type="submit"` would allow to proceed to the next modal by pressing `Enter` while the focus is on the input field.